app/(golems)/golems/skills/[name]/page.tsx

```
@@ -358,6 +358,243 @@ export default async function SkillDetailPage({
+          <div className="space-y-5 lg:sticky lg:top-20">
+            {/* Quick Install */}
+            <div className="rounded-xl border border-[#2dd4a826] bg-[#14120e]/90 p-5">
+              <h3 className="mb-3 text-sm font-bold text-[#2dd4a8]">
+                Quick Install
+              </h3>
+              <div className="group relative mb-3">
+                <CopyButton text={installCommand} />
+                <pre
+                  className="scrollbar-none overflow-x-auto rounded-lg border border-[#2dd4a81a] bg-black/40 p-3 font-mono text-[0.75rem] leading-relaxed text-[#2dd4a8]"
+                  aria-label={`Install command: ${installCommand}`}
+                >
```

Biome (2.4.6)

[error] The ARIA attribute 'aria-label' is not supported by this element.

(lint/a11y/useAriaPropsSupportedByRole)